extension/src/experiments/model/index.ts

```diff
@@ -403,6 +410,23 @@ export class ExperimentsModel extends ModelWithPersistence {
     return this.isBranchesView
   }
 
+  public selectBranchesToShow(branches: string[]) {
```

Comment:
Should this be `setBranchesToShow`? When I think `select`, I think it's going to open a quick pick or something similar.